gnc/navigator_perception/include/navigator_vision_lib/model.h

```diff
@@ -0,0 +1,16 @@
+#ifndef MODEL_H
+#define MODEL_H
+#include <string>
+#include <vector>
+
+class Model
```

Comment:
I'm going to keep the name of the file to model.h / model.cc to aviod merging problems. I will change it to ModelPerception after all this crazy merging is done.